janusgraph-codepipelines-ci/tp-pipe.yml

```diff
@@ -1,13 +1,35 @@
+# Copyright 2017 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# In general, set the timeouts to twice the usual execution time for the compute type.
+# 480 minutes is the maximum build time now.
+#
+defaultComputeImage: docker:dind
+defaultComputeType: BUILD_GENERAL1_LARGE
+defaultPrivilegedMode: true
 pipelines:
 - name: jtp1
   parallelBuildActions:
   - name: e-h-l-s-tp
```

Comment:
You can remove this pipeline since none of these modules have TinkerPop tests and you're excluding the default tests.